### PR TITLE
Update Helm binary

### PR DESCRIPTION
**Pull request by a-kad**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -181,18 +184,18 @@ binaries: helm download-chart ## Download binaries
 
 .PHONY: helm
 helm: bin-dir
```
Comment (TylerGillson) [inline Makefile]: All of this logic for installing Helm locally can be deleted.

File: Dockerfile [hunk 2]
```diff
@@ -1,17 +1,22 @@
...
+ENV BIN_TYPE=${BIN_TYPE:-palette}
+
+ARG HELM_CHART=./bin/vcluster-0.18.1.tgz
 ARG TARGETOS
 ARG TARGETARCH
 
 WORKDIR /workspace
 
 # Copy binaries
 COPY ${HELM} helm
```
Comment (TylerGillson) [inline Dockerfile:16]: We don't need this line anymore.

File: Makefile
Comment (TylerGillson) [inline Makefile:1]: Need to add the following:
`TARGETARCH ?= amd64`
...
Add to docker build args: `--platform linux/${TARGETARCH}`

**Review by TylerGillson — CHANGES_REQUESTED** [review]

File: Makefile [hunk 3]
```diff
@@ -6,14 +6,15 @@
...
+TARGETARCH ?= amd64
 BUILDER_3RDPARTY_VERSION ?= $(shell echo $(PROD_VERSION) | cut -d. -f1,2)
 BUILD_DATE:=$(shell date +%Y%m%d)
 IMG_NAME ?= cluster-api-virtual-controller
 # IMG_URL ?= gcr.io/spectro-images-public/release/cluster-api-virtual/
 IMG_URL ?= gcr.io/spectro-common-dev/${USER}/cluster-api-virtual
 IMG_TAG ?= v0.1.3-spectro-${BUILD_DATE}
 IMG ?= $(IMG_URL)/$(IMG_NAME):$(IMG_TAG)
-BUILD_ARGS = --build-arg CRYPTO_LIB=${FIPS_ENABLE} --build-arg BUILDER_GOLANG_VERSION=${BUILDER_GOLANG_VERSION} --build-arg BUILDER_3RDPARTY_VERSION=${BUILDER_3RDPARTY_VERSION}
+BUILD_ARGS = --build-arg CRYPTO_LIB=${FIPS_ENABLE} --build-arg BUILDER_GOLANG_VERSION=${BUILDER_GOLANG_VERSION} --build-arg BUILDER_3RDPARTY_VERSION=${BUILDER_3RDPARTY_VERSION} --platform linux/${TARGETARCH}
```
Comment (TylerGillson) [inline Makefile]: Sorry Ahmed - the `--platform` arg is actually not a build arg. See https://github.com/spectrocloud/palette-cli/blob/main/Makefile#L174 for an example.

**Review by TylerGillson — CHANGES_REQUESTED** [review]

File: Makefile (hunk 3 above)
Comment (a-kad) [reply Makefile]: Can you please check now @TylerGillson ?

**Review by TylerGillson — APPROVED** [review]
lgtm

**Pull request merged**